main_test.go

```diff
@@ -150,6 +155,98 @@ func TestTransferTokensToAccountActions(t *testing.T) {
 	})
 }
 
+func TestTransferTokensToAccountActionsWith20Signers(t *testing.T) {
+	var transferTokenActionUUID uint64
+	var transferNFTActionUUID uint64
+
+	otu := NewOverflowTest(t)
+	otu.MintFlow("signer1", TransferAmount)
+	otu.SetupTreasury("treasuryOwner", MaxSigners, uint64(MaxThreshold))
+	otu.SendFlowToTreasury("signer1", "treasuryOwner", TransferAmount)
+	otu.CreateNFTCollection("account")
+	otu.MintNFT("account")
+
+	otu.CreateNFTCollection("treasuryOwner")
+	otu.SendCollectionToTreasury("treasuryOwner", "treasuryOwner")
+	otu.SendNFTToTreasury("account", "treasuryOwner", 0)
+
+	t.Run("Signers should be able to propose a transfer of fungible tokens out of the Treasury", func(t *testing.T) {
+		otu.ProposeFungibleTokenTransferAction("treasuryOwner", MaxSigners[0], RecipientAcct, TransferAmount)
+	})
+
+	t.Run("Signers should be able to sign to approve a proposed action to transfer tokens", func(t *testing.T) {
+		// Get first ID of proposed action
+		actions := otu.GetProposedActions("treasuryOwner")
+		keys := make([]uint64, 0, len(actions))
+		for k := range actions {
+			keys = append(keys, k)
+		}
+		transferTokenActionUUID = keys[0]
+
+		// Each signer submits an approval signature
+		for _, signer := range MaxSigners {
+			otu.SignerApproveAction("treasuryOwner", transferTokenActionUUID, signer)
+		}
+
+		// Assert that the signatures were registered
+		signersMap := otu.GetVerifiedSignersForAction("treasuryOwner", transferTokenActionUUID)
+		for _, signer := range MaxSigners {
+			assert.True(otu.T, true, signersMap[otu.GetAccountAddress(signer)])
+		}
+	})
+
+	t.Run(`A signer should be able to execute a proposed action to transfer tokens once it has received
+		the required threshold of signatures`, func(t *testing.T) {
+		otu.ExecuteAction("treasuryOwner", transferTokenActionUUID)
+
+		// Assert that all funds have been transfered out of the treasury vault
+		treasuryBalance := otu.GetTreasuryVaultBalance("treasuryOwner", FlowTokenVaultID)
+		assert.Equal(otu.T, uint64(0), treasuryBalance)
+
+		// Assert that all funds have been received by the recipient account
+		recipientBalance := otu.GetAccount(RecipientAcct).Balance
+		assert.Equal(otu.T, TransferAmountUInt64+DefaultAccountBalance, recipientBalance)
+	})
+
+	t.Run("Signers should be able to propose a transfer of a non-fungible token out of the Treasury", func(t *testing.T) {
+		// TODO: create collection in one of the signer accounts
+		otu.ProposeNonFungibleTokenTransferAction("treasuryOwner", MaxSigners[0], "account", uint64(0))
```

Comment:
am I reading this right that the recipient is a string of just "account"? Feels like that should fail lol

Reply:
the recipient in this case is `emulator-account`!  Overflow takes the account name to resolve the address, but you can drop the `emulator-` prefix.